student.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require './person'
+
+# Represents a student subclass with a specific specialization.
+class Student < Person
+  def initialize(classroom)
+    super
+    @classroom = classroom
+  end
```

Comment:
Nice job so far👍🏼. but just as required, the Constructor should extend the parent's constructor by adding @classroom and a parameter for it😉. see the suggestion below for a better way to implement it😉.

```suggestion
  def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```

Reply:
@Ibizugbe  The code you provided as a suggestion isn't DRY since you repeated the age, name, and parent_permission parameters. It defeats the purpose of the keyword `super` which comes with all those parameters.